platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/api/ConfigurationRoleUsageIntegrationTest.groovy

```diff
@@ -110,34 +110,6 @@ class ConfigurationRoleUsageIntegrationTest extends AbstractIntegrationSpec impl
         assertUsageLockedFailure('testConf')
     }
 
-    def "can add resolution alternatives to configuration deprecated for resolution"() {
```

Comment:
❓ Do you think these tests are still needed?

Reply:
That makes sense.  Adding a path to migrate out of those multi-role configurations is a good thing to have.